packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx

```diff
@@ -111,8 +130,12 @@ export const AccountsMenu = () => {
         <Wrapper>
             <MenuHeader>
                 <Row>
-                    <AccountSearchBox />
-                    <AddAccountButton data-test="@account-menu/add-account" device={device} />
+                    {!isEmpty && <AccountSearchBox />}
```

Comment:
With this way of getting isEmpty: `const isEmpty = filteredAccounts.length === 0;`

this seems to me that the `AccountSearchBox`  will be hidden when search result is empty. Is that right?

It seems to me that isEmpty shall be `const isEmpty = list.length === 0;` instead. But maybe I am missing something.

Reply:
Yes you are correct, missed this, thanks